desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -227,6 +227,9 @@ def execute(self, notebook, snippet):
         self.interpreter.get('dialect_properties') and self.interpreter['dialect_properties']['trim_statement_semicolon']:
       statement = statement.strip().rstrip(';')
 
+    if 'database' in snippet and snippet['datanse']:
```

Comment:
Obviously wrong `datanse`

Let's add a test, it will break an existing one anyway, so let's update the test too

Reply:
so you are talking for this test https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/connectors/sql_alchemy_tests.py#L251 or creating a new test for `execute`

Reply:
Updating the failing test

Reply:
I have checked that after correction failing test is passing now.